src/Components/PresentationalComponents/CSAwRuleBox/CSAwRuleBox.js

```diff
@@ -16,10 +16,14 @@ import { Stack,
 } from '@patternfly/react-core';
 import { useDispatch } from 'react-redux';
 import { Link } from 'react-router-dom';
-import { InsightsLabel } from '@redhat-cloud-services/frontend-components/components/esm/InsightsLabel';
-import CheckCircleIcon from '@patternfly/react-icons/dist/js/icons/check-circle-icon';
-import OutlinedQuestionCircleIcon from '@patternfly/react-icons/dist/js/icons/outlined-question-circle-icon';
-import ExternalLinkAltIcon from '@patternfly/react-icons/dist/js/icons/external-link-alt-icon';
+import { InsightsLabel } from '@redhat-cloud-services/frontend-components';
+import {
+    CheckCircleIcon,
+    OutlinedQuestionCircleIcon,
+    ExternalLinkAltIcon,
+    AnsibeTowerIcon,
```

Comment:
@karelhala I'm a bit confused. I thought using the full path helps with the bundle size. What did it change?

Reply:
It helps, but we are using transform plugin for that https://github.com/RedHatInsights/vulnerability-ui/blob/master/babel.config.js#L68

The reason why transform plugin is favorited is that if some path changes, one single change is required or (more targeted to this repo) when releasing different envs (esm, umd, cjs) we can use said build type and not mix cjs with esm.